docs/destinations/forwarding-events/elasticsearch/index.md

```diff
@@ -24,33 +29,33 @@ First off, install and set up Elasticsearch version 7.x or 6.x. For more informa
 
 Elasticsearch keeps a lot of files open simultaneously so you will need to increase the maximum number of files a user can have open. To do this:
 
-```bash
-sudo vim /etc/security/limits.conf
-```
+<CodeBlock language="bash">{
```

Comment:
You don’t need to change all code blocks! But I understand if it’s in pursuit of consistency.